packages/flutter_markdown/lib/src/markdown_raw.dart

```diff
@@ -119,10 +133,23 @@ class _MarkdownBodyRawState extends State<MarkdownBodyRaw> {
     MarkdownStyleRaw markdownStyle = config.markdownStyle ?? config.createDefaultStyle(context);
     SyntaxHighlighter syntaxHighlighter = config.syntaxHighlighter ?? new _DefaultSyntaxHighlighter(markdownStyle.code);
 
-    _cachedBlocks = _blocksFromMarkup(config.data, markdownStyle, syntaxHighlighter);
+    _linkHandler = new _LinkHandler(config.onTapLink);
+
+    // TODO: This can be optimized by doing the split and removing \r at the same time
+    List<String> lines = config.data.replaceAll('\r\n', '\n').split('\n');
+    md.Document document = new md.Document();
+
+    _Renderer renderer = new _Renderer();
+    _cachedBlocks = renderer.render(document.parseLines(lines), markdownStyle, syntaxHighlighter, _linkHandler);
+  }
```

Comment:
What happens if you change the `config.data` for the `MarkdownBodyRaw` widget?  It looks like you only process the data during initState and not during didUpdateConfig.  This is a pre-existing issue and probably better fixed in a separate patch.